code/modules/modular_computers/file_system/programs/app_presets.dm

```diff
@@ -234,6 +234,27 @@
 	)
 	return _prg_list
 
+/datum/modular_computer_app_presets/command/teleporter
+	name = "command_teleporter"
+	display_name = "Command - Teleporter"
+	description = "Contains the most common command programs and has a special teleporter control program loaded."
+	available = FALSE
+
```

Comment:
If you want to place that laptop in a public area, dont use a command subtype.

Reply:
as discussed, this isn't in a public area